cookbooks/galera/recipes/server.rb

```diff
@@ -57,7 +62,7 @@
   shell "/bin/false"
 end
 
-galera_config = data_bag_item('s9s_galera', 'config')
+galera_config = node[:galera_config]
```

Comment:
If you want to be upstream-friendly, you should make the use of attribute-based configuration optional, like:
galera_config = node[:galera_config] || data_bag_item(blah...)